### PR TITLE
YDA-5877 improve admin page UI consistency

### PR DESCRIPTION
**Pull request by FuHsinyu**
The change primarily includes the following:
1. Combined three features into one card and let bootstrap grid manages UI layouts
2. improve / add unique id for each button
3. Functionality remains unchanged

For the updated UI tests see the associated PR: https://github.com/UtrechtUniversity/yoda-ruleset/pull/495

For the new UI screenshot, see
image

### CODE REVIEW

File: admin/templates/admin/admin.html [hunk 1]
```diff
@@ -13,109 +13,105 @@
...
+                <!-- Updated content will be displayed here -->
+            </div>
+            <div class="modal-footer">
+                <div class="row">
+                    <div class="col-sm-12">
+                        <fieldset>
+                            <input type="checkbox" class="confirm-conditions" id="checkbox-confirm-conditions">
+                            <label for="checkbox-confirm-conditions">Please confirm that you agree with the
+                                above</label>
+                        </fieldset>
```
Comment (claravox) [inline admin/templates/admin/admin.html]: Maybe you can extract this modal to a template file that we can `include` both in browse.html in the vault and here. But if it turns out to be too much work this is good as well.

Comment (FuHsinyu) [reply admin/templates/admin/admin.html]: This is a nice suggestion. I'll look it up

Comment (claravox) [reply admin/templates/admin/admin.html]: Nice, can you make the Confirm button dismiss the modal but only when in the admin page?

**Review by claravox — APPROVED** [review]
LGTM!

**Pull request merged**